QUICKSTART.md

```diff
@@ -0,0 +1,45 @@
+# Kind
+This guide was created to help new users of Kubeflow install and test Kubeflow Helm Charts locally based on the kind.
+
+## Requirements
+* kind v0.25.0 or above
```

Comment:
`v0.25.0` is just a month old, we could definitely run on an older version. Maybe we can write "tested on `v0.25.0` but should work with older versions as well"?